qemu/tests/driver_load.py

```diff
@@ -44,15 +45,28 @@ def run(test, params, env):
     driver_id = driver_id[0]
     if params["os_type"] == "windows":
         driver_id = '^&'.join(driver_id.split('&'))
+        devcon = params.get("devcon")
+        devcon_cli = devcon % utils_misc.get_winutils_vol(session)
+        session.cmd(devcon_cli)
     driver_load_cmd = driver_load_cmd.replace("DRIVER_ID", driver_id)
     driver_unload_cmd = driver_unload_cmd.replace("DRIVER_ID", driver_id)
 
     for repeat in range(0, int(params.get("repeats", "1"))):
         error.context("Unload and load the driver. Round %s" % repeat,
                       logging.info)
-        session.cmd(driver_unload_cmd)
+        error.context("unload CLI %s" % driver_unload_cmd,
```

Comment:
1. suggest to use logging.info("unload ...") instead of error.context as it's a cmd info not a step
2. something wrong in the driver_id
" unload CLI C:\devcon.exe disable @'C "